src/verify.c

```diff
@@ -68,6 +68,7 @@ static const struct option prog_opts[] = {
 	{ "quick", no_argument, 0, 'q' },
 	{ "force", no_argument, 0, 'x' },
 	{ "statedir", required_argument, 0, 'S' },
+	{ "verify-sign", no_argument, 0, 'V' },
```

Comment:
nitpick on grammar. proper term would be "verify-sig" or "verify-sigs" if multiple possible.